CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ You can propose an article as a [GitHub issue](https://github.com/tpf/perldotcom
 
 	% make new
 
-You can then check your article by starting a local server (using [Hugo 0.59](https://gohugo.io/news/0.59.0-relnotes/) and no other version):
+You can then check your article by starting a local server using [Hugo](https://gohugo.io/). Note that v0.59 of Hugo was previously recommended, however the repo here has confirmed to be working with versions as recent as v0.126. To start the server run:
```

Comment:
I think the issue may not be in the build, but rather in the render. That is, I don't think anyone has taken the time to prove that the render is correct on later versions. Having said that, we do use a pinned version to deploy and as long as the new articles don't do anything unorthodox, this should probably be fine.

Reply:
I'll add this detail to the update 👍